ftw/globalstatusmessage/tests/test_controlpanel.py

```diff
@@ -140,7 +147,8 @@ def test_receiver(self, browser):
         browser.login().open(
             self.portal,
             view='@@global_statusmessage_config_receiver',
-            data={'jsondata': self.payload})
+            data={'jsondata': self.payload},
+            send_authenticator=HAS_MODERN_PLONE_PROTECT)
```

Comment:
Did you run into an import error on Plone 4 in `ftw.testbrowser` when setting this always to `True`?

I'd prefer that we can set `send_authenticator=True` and `ftw.testbrowser` would detect wether it is necessary or not 🤔